monitoring/monitoring.go

```diff
@@ -547,11 +563,54 @@ func getServiceRAMUsage(uid int32) (ram uint64, err error) {
 	return ram, nil
 }
 
-// getServiceDiskUsage returns service disk usage in bytes
-func getServiceDiskUsage(path string, uid, gid uint32) (diskUse uint64, err error) {
-	if diskUse, err = platform.GetUserFSQuotaUsage(path, uid, gid); err != nil {
+func getListProcesses() (processes []processInterface, err error) {
```

Comment:
Rename `getListProcesses` to `getProcesses` or `getProcessesList`.

Reply:
use getProcessesList

Reply:
use getProcessesList